Source/WebKit/WebKit.xcodeproj/project.pbxproj

```diff
@@ -18909,7 +18909,9 @@
 		};
 		5C400E6A29DB8AB500446F6F /* PBXTargetDependency */ = {
 			isa = PBXTargetDependency;
-			platformFilter = ios;
+			platformFilters = (
+				ios,
+			);
```

Comment:
Seems like noise, maybe on account of your Xcode version being newer. Could you delete?

Reply:
Done!